app/models/attachment.rb

```diff
@@ -54,6 +54,10 @@ def self.parliamentary_sessions
     end
   end
 
+  def is_official_document?
+    is_command_paper? || is_act_paper?
+  end
```

Comment:
Should this have a test?

Reply:
I don't think so, both `is_command_paper` and `is_act_paper` have tests and this is super simple logic. If you take a look at https://github.com/alphagov/whitehall/blob/master/test/unit/attachable_test.rb#L209 you can see a similar pattern; a `has_official_document` method exists that does the same thing but isn't tested because `has_command_paper` and `has_act_paper` are.

Happy to write one if you'd like though :)